### DISCUSSION TITLE
Project's future

### DISCUSSION THREAD

**Discussion by gshpychka** [nvimdev/lspsaga.nvim#216] [General]
I can see that @glepnir hasn't commited anything to github since April. I hope everything is alright.

WIth that said, is there anyone actively working on a fork? Seems like this project might be dead.

**Comment by glepnir** [comment]
because i&nbsp; am in hospital .maybe next weekend I can go home .need doctor allow that

--------------原始邮件--------------
发件人："Glib Shpychka ***@***.***&gt;;
发送时间：2021年6月24日(星期四) 下午3:40
收件人："glepnir/lspsaga.nvim" ***@***.***&gt;;
抄送："Raphael ***@***.***&gt;;"Mention ***@***.***&gt;;
主题：[glepnir/lspsaga.nvim] Project's future (#216)
-----------------------------------

 

 
I can see that @glepnir hasn't commited anything to github since April. I hope everything is alright.
 
WIth that said, is there anyone actively working on a fork? Seems like this project might be dead.

**Reply by Joakker** [reply]
Hope you get better soon my friend

**Reply by thedelchop** [reply]
Thank you for all your work and here's hoping for a speedy recovery!

**Reply by rgnkn** [reply]
Get well soon :-)

**Reply by siduck** [reply]
get well soon buddy!

**Reply by Vanderscycle** [reply]
Hey, get well soon. If you know a list of issues and improvements that you'd like done you could list them and I'll try to help (I am sure a few people will too).

**Reply by Bacbia3696** [reply]
Hope best wish for you !

**Reply by chriztheanvill** [reply]
Best wishes.
Saludos desde Mexico :D

**Reply by Near2U0** [reply]
一切顺利

**Comment by Mattio-cmd** [comment]
You good bro? @glepnir

**Comment by par4m** [comment]
Just noticed a [recent commit](https://github.com/glepnir/nvim/commit/8783a2a170dedd0a2adf27a32690b74bea56a330) by @glepnir, hope they're alright now.
